todo-backend/README.adoc

```diff
@@ -498,6 +617,8 @@ $ helm show readme jboss-eap/eap-xp3
 ----
 endif::[]
 
+// Testing on Openshift
```

Comment:
The testing section should be present in the Readme after the `Environment variables for PostgreSQL` since we need to wait until the QS is deployed first.

Reply:
It's OK now, thanks.